src/page-weekly-snapshot/WeeklySnapshotPage.tsx

```diff
@@ -23,8 +110,24 @@ const WeeklySnapshotPage: React.FC = () => {
             <Loading />
           ) : (
             <WeeklySnapshotContainer>
-              <SnapshotPage header="COVID-19 Report" subheader>
-                YTD Summary and impact report
+              <SnapshotPage
+                image={stateImage}
+                header="COVID-19 Report"
+                subheader
+              >
+                <br />
+                <BorderDiv />
+                <TextContainer>
+                  <LeftHeading marginTop={"0px"}>
+                    {stateName} / {todayFormatted} / Year-to-date projected
```

Comment:
What does this render as if `stateName` remains undefined after the logic above? "undefined / ..." or "null / ..." or just " / ..."?

Reply:
I see now that it's that third one, the leading slash. I think that's acceptable but just want to flag in case it's not a conscious decision on your part.

Reply:
Thanks for flagging! That was intentional on my part; it would be relatively simply to not render this entire section if `stateName` is null, but this seemed sufficient to me.